.bazelrc

```diff
@@ -1,12 +1,17 @@
 try-import %workspace%/.bazelrc.user
 
-build --symlink_prefix=_bazel/ --ui_event_filters=-DEBUG --define gotags=bazel,gss --experimental_proto_descriptor_sets_include_source_info --incompatible_strict_action_env --incompatible_enable_cc_toolchain_resolution
-test --config=test --experimental_ui_max_stdouterr_bytes=10485760
+build --define gotags=bazel,gss
+build --experimental_proto_descriptor_sets_include_source_info
+build --incompatible_strict_action_env --incompatible_enable_cc_toolchain_resolution
+build --symlink_prefix=_bazel/ --experimental_no_product_name_out_symlink
+test --config=test --experimental_ui_max_stdouterr_bytes=10485760 --experimental_no_product_name_out_symlink
```

Comment:
Nit: `test` inherits from `build`, so `--experimental_no_product_name_out_symlink` is implied.